source/agora/consensus/data/Transaction.d

```diff
@@ -268,3 +294,46 @@ public bool getSumOutput (const Transaction tx, ref Amount acc)
             return false;
     return true;
 }
+
+unittest
+{
+    import vibe.data.json;
+    Transaction old_tx = Transaction(
+        TxType.Data,
+        [Input(Hash.init, 0)],
+        [Output.init],
+        [1,2,3]
+    );
+    auto json_str = old_tx.serializeToJsonString();
```

Comment:
I tried to express it in hexadecimal when expressed in JSON.
However, I finally decided that it was better to express it in an array in terms of performance.